var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -55,33 +56,38 @@ class PyScipy(PythonPackage):
     # https://github.com/mesonbuild/meson-python/pull/167
     depends_on("py-build", when="@1.9:", type="build")
 
-    depends_on("py-meson-python@0.9:", when="@1.9.2:", type="build")
+    depends_on("py-meson-python@0.11", when="@1.10:", type="build")
```

Comment:
I'd like to keep the deps as close as possible to what is documented in pyproject.toml. We're currently disabling dep checks for pip but when we move to build/installer I think it will complain if the versions don't match. So once 1.10.1 is released I'll update this.